views/registration.njk

```diff
@@ -9,7 +9,7 @@
 {% endblock %}
 
 {% block customCSS %}
-<link rel="stylesheet" href="stylesheets/registration.css">
+<link rel="stylesheet" href="../stylesheets/registration.css">
```

Comment:
Since the public folder is static. You can just do `/stylesheets`

Reply:
Travis CI failed on me when I had that before... it couldn't locate it. I believe it's because of the query string (it adds an additional / to the route)? I'll try again and see what happens.

Reply:
In your original one you did not have a preceding `/` and it was just `stylesheets/...`

Reply:
I did have a leading '/' but I'm thinking I just misread the Travis results. I think it didn't find the stylesheets because (at the time) I hadn't updated "test-pages" yet. So, Travis was sending a get request to the '/registration' route (which no longer existed) that had been replaced by '/registration/:badge'. It is working with the hard-coded test badge number now.